packages/search/addon/components/navi-search-result/asset.hbs

```diff
@@ -0,0 +1,10 @@
+{{! Copyright 2020, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. }}
+<ul class='navi-search-results'>
+  {{#each @data as |result|}}
+    <li class='navi-search-result__asset'>
```

Comment:
Maybe delegate this one step further so that it can be easily changed

```hbs
{{component result.whateverType result=result}}
```

That way we can change just how reports/dashboards/etc look

Reply:
This Asset component isn't supposed to be a general component, it'll only handle displaying report and dashboard results in a certain way

Reply:
Right now it has no styling, I'll add all the styling with the search bar PR
<img width="172" alt="Screen Shot 2020-03-04 at 3 51 12 PM" src="https://user-images.githubusercontent.com/1649445/75926357-06939d80-5e30-11ea-8d3a-b5414f09eddb.png">